### PR TITLE
Update focus-appearance.html

### PR DESCRIPTION
**Pull request by mbgower**
attempts to address comments circa https://github.com/w3c/wcag/issues/2706#issuecomment-1422857287

### CODE REVIEW

**Comment by alastc** [comment]
I'm not sure about the `target="blank"` additions, if some do have that, and some don't, we should probably mark those that open a new window.

**Comment by mbgower** [comment]
> I'm not sure about the target="blank" additions, if some do have that, and some don't, we should probably mark those that open a new window.

I'm fine with how you want to proceed, but in a document this big, when your existing location is overwritten by a new page as you active a link, it makes it a lot harder to reorient yourself when you reload the page.

**Comment by mbgower** [comment]
> we should probably mark those that open a new window.

I would have thought that is an automatic thing across all of w3c if that's the case? maybe there's not

File: understanding/22/focus-appearance.html [hunk 1]
```diff
@@ -434,15 +434,16 @@ <h3>First exception: the focus indicator cannot be adjusted by the author</h3>
...
+        <p style="note">Altering the <code>body</code> element's <code>background-color</code> attribute <em>is</em> one way of altering the pixels directly adjacent to the indicator in most implementations. However, specifying a value of white (<code>#FFFFFF</code>) does not nullify this exception since, as established in the third note of the <a href="https://www.w3.org/TR/WCAG22/#dfn-contrast-ratio" target="_blank">contrast ratio definition</a>, the default ("unspecified") color is assumed to be white.</p>
         
-            <p>[Future image set: a button with a default light blue focus indicator; the same buton and focus indicator on a light blue page background]</p>
+            <p>[Future image set: a button with a default light blue focus indicator on a white page; the same buton and focus indicator on a light blue page background]</p>
```
Comment (patrickhlauke) [inline understanding/22/focus-appearance.html:444]: there's an existing typo worth fixing: "buton" > "button"

**Comment by patrickhlauke** [comment]
i know this is closed, but...looks reasonable to me either way

**Comment by alastc** [comment]
Approved: https://www.w3.org/2023/02/28-ag-minutes#r08

**Pull request merged**